homeassistant/components/lg_netcast/media_player.py

```diff
@@ -205,7 +217,10 @@ def media_image_url(self):
 
     def turn_off(self):
```

Comment:
If there is always a `turn_off` action to perform why should the integration only return `SUPPORT_TURN_OFF` if an action script has been defined?

Reply:
Well seen, thx.
Just figured, that "turn off" is actually always defined as a supported feature in line 50. So it seems not necessary to handle this specifically. 
I'm gonna revert this to previos commit state.